### PR TITLE
Code review edit

### PR DESCRIPTION
**Pull request by mllrdev-jerry-bayoneta**

### CODE REVIEW

File: src/composables/feed/useDeleteById.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    await client({
+      method: 'DELETE',
+      url: `posts/${id}`
+    }).then(() => {
+      return true
+    })
+    .catch(() => {
+      return false
+    })
+
```
Comment (KarlZablan) [inline src/composables/feed/useDeleteById.ts]: remove extra whitespaces

File: src/composables/feed/useEditbyId.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      method: 'PATCH',
+      url: `posts/${editedPost.id}`,
+      data: editedPost
+    }).then(() => {
+      return true
+    })
+    .catch(() => {
+      return false
+    })
+
```
Comment (KarlZablan) [inline src/composables/feed/useEditbyId.ts]: remove extra whitespace

File: src/shared/component/post.vue [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+import useDeleteById from '@/composables/feed/useDeleteById'
+import Modal from '@/shared/component/modal.vue'
+
+export default defineComponent({
+  name: "Post",
+  components: {
+    Modal,
+  },
+  props:{
+    id: {
```
Comment (KarlZablan) [inline src/shared/component/post.vue]: It looks like these props are properties of INewsfeed. How about using INewsfeed as a prop.

File: src/shared/component/post.vue [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    const { deletePostById } = useDeleteById();
+    const news = props;
+    const router = useRouter();
+    const read = ref(false)
+    function deletePost(id: number) {
+      if (confirm(`delete?`)) {
+        deletePostById(id);
+      }
+        location.reload();
+
```
Comment (KarlZablan) [inline src/shared/component/post.vue]: remove extra whitespace

File: src/views/Edit.vue [hunk 5]
```diff
@@ -1,82 +1,77 @@
...
-      editedPost.updatedAt = this.date();
-      newsFeedServices.editNewsFeedById(editedPost, id);
-      router.push({
-        name: RouteName.FEED
-      })
+    async function applyEditOnNewsFeed(editedPost: INewsFeed){
+      editedPost.updatedAt = date.toLocaleDateString()
+      const res = await editById(editedPost)
+        if(res){
+          console.log('jer')
```
Comment (KarlZablan) [inline src/views/Edit.vue]: remove all console.log

**Review by KarlZablan — CHANGES_REQUESTED** [review]

File: src/composables/feed/useDeleteById.ts (hunk 1 above)
Comment (mllrdev-jerry-bayoneta) [reply src/composables/feed/useDeleteById.ts]: deleted whitespace

File: src/composables/feed/useEditbyId.ts (hunk 2 above)
Comment (mllrdev-jerry-bayoneta) [reply src/composables/feed/useEditbyId.ts]: deleted whitespace

File: src/shared/component/post.vue (hunk 3 above)
Comment (mllrdev-jerry-bayoneta) [reply src/shared/component/post.vue]: used INewsfeed as prop

File: src/shared/component/post.vue (hunk 4 above)
Comment (mllrdev-jerry-bayoneta) [reply src/shared/component/post.vue]: deleted whitespace

File: src/views/Edit.vue (hunk 5 above)
Comment (mllrdev-jerry-bayoneta) [reply src/views/Edit.vue]: erased console.log

**Pull request merged**